frontend/express/public/core/events/javascripts/countly.details.views.js

```diff
@@ -333,16 +376,16 @@
             eventKey: eventKey && eventKey !== "undefined" ? eventKey : undefined
```

Comment:
do you actually need this check here? 😄 
you could have simply written -
eventKey: eventKey

Reply:
Here undefined is coming as a string("undefined") and hence the explicit check.

Reply:
i believe it comes from here - https://github.com/Countly/countly-server/pull/1972/files/7a57786ea461153ad233c4c52296e9ed340bfb24#diff-30799c866b71bbc51ee3ee16c468adf1591ffb063b4158e86bd1c081ee998e03R15

can params.name be "undefined" ?

Reply:
but isn't params.name indirectly coming from the table rows, and it can only be "undefined" if the event is named undefined.
and if the event is named undefined, your check above will not allow that event.

Reply:
am i missing something?
can this be triggered from somewhere else?

Reply:
Hello, so this check is not when coming from tables. But in case user  directly lands on compare and clicks on event stats :)

Reply:
i think i mentioned this in some other thread here, but when clicking on event stats, you should open /analytics/events
and therefore this route (the one with :key) should not be triggered

Reply:
as per our discussion this is expected in the upcoming pr.
keeping a note.